docs/esa/jwst/jwst.rst

```diff
@@ -430,7 +430,7 @@ Query without saving results in a file:
     Output file: 1655977738493O-result.vot.gz
     Results: None
   >>> result = job.get_results()
-  >>> result
+  >>> result  # doctest: +IGNORE_OUTPUT
```

Comment:
it maybe worth fixing these, as I see the difference is some reformatting of the output for the proposal id, (though it can be a rabbit hole if the other examples below this one are also failing)

Reply:
I think it's too fragile, I see different results from the test infrastructure not because the output table is different but because the number of lines shown is.

Reply:
If the lines are identical, the difference of how many of them being show can be hidden in an ellipsis